terragrunt/aws/glue/etl/bes/crm/process_salesforce.py

```diff
@@ -50,19 +50,35 @@
 )
 
 # Rename conflicting columns
-df_transformed = df_transformed.rename(columns={"Name_x": "AccountName", "CreatedDate_x": "AccountCreatedDate",
-                                       "CreatedDate_y": "OpportunityCreatedDate", "Id_y": "OpportunityId", "Name_y": "OpportunityName"})
+df_transformed = df_transformed.rename(
```

Comment:
curious what extension and python formatter did you use?

Reply:
I used this one:
https://pypi.org/project/black/

It's pretty common across our repos but there are other formatters in use I think.